socialforce/simulator.py

```diff
@@ -39,11 +41,11 @@ def __init__(self, initial_state, ped_space=None, delta_t=0.4, tau=0.5):
             self.state = np.concatenate((self.state, np.expand_dims(tau, -1)), axis=-1)
 
         # potentials
-        self.V = PedPedPotential(self.delta_t)
+        self.V = PedPedPotential(self.delta_t, v0, sigma)
```

Comment:
Make this `self.V = ped_ped or PedPedPotential(self.delta_t)`.

Reply:
I would like to keep (v0, sigma) so that one can provide them as arguments to the simulator initializer for hyper parameter tuning. Is there any other way to make this happen? 
If I remove (v0, sigma), one cannot assign different values to these hyper parameters.

Reply:
you can provide `ped_ped=PedPedPotential(v0, sigma)` and also `ped_ped=MyNewFunkyPedPedPotential( parameter1, parameter2, parameter3)`.